front/tests/010-op-route-tab.spec.ts

```diff
@@ -1,19 +1,19 @@
 import { test } from '@playwright/test';
 import { v4 as uuidv4 } from 'uuid';
 
-import type { Infra, Project, Scenario, Study, Timetable } from 'common/api/osrdEditoastApi';
+import type { Infra, Project, Scenario, Study, TimetableResult } from 'common/api/osrdEditoastApi';
 
 import scenarioData from './assets/operationStudies/scenario.json';
 import HomePage from './pages/home-page-model';
 import OperationalStudiesPage from './pages/operational-studies-page-model';
 import ScenarioPage from './pages/scenario-page-model';
-import { getProject, getStudy, postApiRequest, getInfra } from './utils/index';
+import { getInfra, getProject, getStudy, postApiRequest } from './utils/index';
 
 let smallInfra: Infra;
 let project: Project;
 let study: Study;
 let scenario: Scenario;
-let timetable: Timetable;
+let timetable_result: TimetableResult;
```

Comment:
When possible, we don't use snake case in front :)

```suggestion
let timetableResult: TimetableResult;
```